web/sites/default/config/core.entity_view_display.node.event.teaser.yml

```diff
@@ -1,4 +1,4 @@
-uuid: 68d1af92-417a-4d95-aaa0-a8c92695e869
+uuid: b0424817-8237-4565-847b-4192b437023b
```

Comment:
Do you know why this UUID changed? AFAIK that doesn't really affect anything, but I was curious why it changed.

Reply:
I didn't do anything special to that display except add the featured image. 🤷